testsuite/features/support/remote_node.rb

```diff
@@ -327,9 +328,9 @@ def client_public_ip
 
   # Extract the OS version and OS family
   # We get these data decoding the values in '/etc/os-release'
-  def get_os_version
-    os_family_raw, code = run('grep "^ID=" /etc/os-release', check_errors: false)
-    os_family_raw, code = run('sw_vers --productName', check_errors: false) if code.nonzero?
+  def get_os_version(run_in_container: true)
+    os_family_raw, code = run('grep "^ID=" /etc/os-release', run_in_container: run_in_container, check_errors: false)
```

Comment:
Where does `run_in_container` come from? I cannot find it here in the PR.

Reply:
And apparently our test do not know either:

```bash
PXE boot MAC address variable empty
unknown keyword: :run_in_container (ArgumentError)
/testsuite/features/support/remote_node.rb:102:in `run'
/testsuite/features/support/remote_node.rb:332:in `get_os_version'
/testsuite/features/support/remote_node.rb:49:in `initialize'
/testsuite/features/support/remote_nodes_env.rb:36:in `new'
/testsuite/features/support/remote_nodes_env.rb:36:in `get_target'
/testsuite/features/support/commonlib.rb:664:in `new_api_client'
```

Reply:
Hmmm, let me look for typos

Reply:
Fixed, thanks for the catch